docs/ref/models/fields.txt

```diff
@@ -1157,6 +1157,35 @@ you can use the name of the model, rather than the model object itself::
         # ...
         pass
 
+Note that relationships defined this way on
+:ref:`abstract models <abstract-base-classes>` are only resolved when the
+model is subclassed as concrete model and are not relative to the abstract
+model's ``app_label``::
+
+    from django.db import models
+
+    class AbstractCar(models.Model):
+        manufacturer = models.ForeignKey(
+            'Manufacturer',
+            on_delete=models.CASCADE,
+        )
+
+        class Meta:
+            app_label = 'products'
+            abstract = True
+
+    class Manufacturer(models.Model):
+        # ...
+
+        class Meta:
+            app_label = 'production'
+
+    class Car(AbstractCar):
+        class Meta:
+            app_label = 'production'
+
+        # Car.manufacturer will point to `production.Manufacturer` here.
```

Comment:
IMO it makes sense to explain it thoroughly in one place, and link to that explanation in the other.

I think a rewrite of this whole section could deliver a clearer explanation than inserting a paragraph, but we shouldn't let that hold up the patch – I would be happy to give that a crack after this is merged. I think we could introduce the three types of string reference more explicitly, give them each names (_recursive_, _relative_, and _absolute_, for example), then go on to explain how they work in an abstract context.